<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>402</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Accord</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-color: #dce3ef;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="plusButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>30</width>
             <height>30</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#plusButton {
	background-color: transparent;
	border-image: url(:images/plus.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="minusButton">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>0</y>
             <width>30</width>
             <height>30</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#minusButton {
	background-color: transparent;
	border-image: url(:images/minus.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QListWidget" name="playListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>40</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">#playListWidget {
	background-color: white;
	border: none;
}
</string>
          </property>
          <property name="currentRow">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="curAudioListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>60</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#curAudioListWidget {
	background-color: white;
	border: none;
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="timeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#timeSlider {
	padding-top: -4px;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="volumeMainBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QGroupBox" name="volumeBox">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>10</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <property name="title">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="volumeButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>5</y>
             <width>40</width>
             <height>40</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#volumeButton {
	background-color: transparent;
	border-image: url(:images/volume.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#groupBox {
	background-color: transparent;
	background: none;
	border: none;
	background-repeat: none
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="nextButton">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>15</y>
             <width>40</width>
             <height>20</height>
            </rect>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#nextButton {
	background-color: transparent;
	border-image: url(:images/next.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="prevButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>15</y>
             <width>40</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#prevButton {
	background-color: transparent;
	border-image: url(:images/previous.png);
	background: none;
	border: none;
	background-repeat: none;
}

#prevButton:hover {
	background-color: white;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QGroupBox" name="playPauseBox">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>0</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="title">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loopPlaylistButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Loop</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="settingsBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="settingsButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>5</y>
             <width>40</width>
             <height>40</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#settingsButton {
	background-color: transparent;
	border-image: url(:images/settings.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
